content/concept-xrp-ledger-intro.md

```diff
@@ -0,0 +1,88 @@
+# XRP Ledger Introduction
+
+The **XRP Ledger** is a decentralized cryptocurrency network. Like all such networks, it's powered by a peer-to-peer server and it enables anyone to send and receive value within the rules of the network. The XRP Ledger is the home of **XRP**, a cryptocurrency which was designed by its creators to bridge the many different currencies preferred by people worldwide. Ripple, the company that stewards development of the XRP Ledger, expects XRP to be a key part of enabling the "Internet of Value": a world in which money moves the way information does today.
```

Comment:
When do we use "cryptocurrency" vs. "digital asset"?

> The **XRP Ledger** is a decentralized cryptographic ledger powered by a network of peer-to-peer servers. It is the home of **XRP**, a digital asset designed to bridge the many different currencies preferred by people worldwide. Ripple, the company that stewards development of the XRP Ledger, expects XRP to be a key part of enabling the _Internet of Value_: a world in which money moves the way information does today.

Above, I've made an attempt to tighten up the wording slightly and avoid using the word "cryptocurrency." Just an idea.

Reply:
That's an interesting thought.

"Cryptocurrency" is a specific subset of "digital asset" and has become fairly well-known at this point. The term "digital asset" rightfully includes things like rewards points that are centrally administered and have very different properties than cryptocurrencies.

I don't think that "cryptographic ledger" is any more intuitive than "cryptocurrency network" and I suspect that our target audience understands the latter term better, but sometimes it's hard to know when I'm in a bubble that happens to look at the terms differently.

Reply:
I attended a Ripple Pitch Training delivered by Ashish that told us to call XRP a "digital asset," not a "cryptocurrency." Is this a rule that we want to apply to the tech docs? What is the difference between a digital asset and a cryptocurrency? Does digital asset more accurately describe XRP? We may need to sync up with PM on messaging.